apiserver/plane/app/views/workspace/recent_visit.py

```
@@ -0,0 +1,24 @@
+from ..base import BaseViewSet
+from plane.app.permissions import allow_permission, ROLE
+
+class UserRecentVisitViewSet(BaseViewSet):
+    model = UserRecentVisit
+
+    def get_serializer_class(self):
+        return WorkspaceRecentVisitSerializer
+    
+    @allow_permission([ROLE.ADMIN, ROLE.MEMBER, ROLE.GUEST], level="WORKSPACE")
+    def list(self, request, slug):
+        user_recent_visits = UserRecentVisit.objects.filter(workspace__slug=slug).filter(entity_name__in=["issue","page","project"])
```

Ruff (0.8.2)

Line too long (132 > 88)

(E501)